components/automate-ui/src/app/components/delete-object-modal/delete-object-modal.component.html

```diff
@@ -1,17 +1,13 @@
 <chef-modal [visible]="visible" (closeModal)="closeEvent()">
   <h2 slot="title">{{objectAction}} {{objectNoun | titlecase}}?</h2>
   <div>
-    <span *ngIf="moreDetails">
-      {{ objectAction | continuous }} {{objectNoun}} {{objectName}} <strong>cannot be undone</strong><br>
-      {{moreDetails}}.
-    </span>
-    <span *ngIf="!moreDetails">
-      {{ objectAction | continuous }} {{objectNoun}} {{objectName}} <strong>cannot be undone</strong>.
-    </span>
+    {{ objectAction | continuous }} {{objectNoun}} {{objectName}}
+    <strong>cannot be undone</strong><span *ngIf="moreDetails"><br>{{moreDetails}}</span>.
   </div>
+  <div *ngIf="errorText" class="error">{{ errorText }}</div>
 
   <div class="options">
-    <chef-button primary caution (click)="deleteObject()">
+    <chef-button primary caution [disabled]="!!errorText" (click)="deleteObject()">
```

Comment:
This is where the action is: disable button when error is displayed.

Reply:
What does the two '!' do here?

Reply:
Shorthand for convert-to-boolean